pygmt/src/ternary.py

```diff
@@ -0,0 +1,64 @@
+"""
+ternary - Plot data on ternary diagrams.
+"""
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    B="frame",
+    C="cmap",
+    F="center",
+    G="fill",
+    JX="projection",
```

Comment:
I don't think `JX="projection"` is a good alias for ternary. For other modules, we usually pass `projection="X10c"`, but in ternary, we need to pass `projection="10c"`. Such inconsistency may confuse users. Can we simply use alias `JX` to `withd` instead?

Reply:
updated